test/unit/test-amp-layout.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {createElementWithAttributes} from '#core/dom';
+import {Layout} from '#core/dom/layout';
+
+describes.realWin('amp-layout', {amp: true}, (env) => {
```

Comment:
Can we move this to `test/unit/builtins/test-amp-layout` to mirror the tree structure?

Reply:
I was thinking the same.
Need to move all 3 builtins though, so figured i should leave it for a followup.